src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectFactory.Windows.cs

```diff
@@ -19,7 +19,7 @@ internal sealed class TdsParserStateObjectFactory
         //private static bool shouldUseLegacyNetorking;
         //public static bool UseManagedSNI { get; } = AppContext.TryGetSwitch(UseLegacyNetworkingOnWindows, out shouldUseLegacyNetorking) ? !shouldUseLegacyNetorking : true;
 
-        public static bool UseManagedSNI { get; } = false;
+        public static bool UseManagedSNI { get; } = true;
```

Comment:
I don't think you want to check in this change.

Reply:
`true` was only for testing only. Rolled back to `false`